packages/datagateway-dataview/cypress/e2e/table/investigations.cy.ts

```diff
@@ -22,7 +22,7 @@ describe('Investigations Table', () => {
 
   it('should disable the hover tool tip by pressing escape', () => {
     // The hover tool tip has a enter delay of 500ms.
-    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    // eslint-disable-next-line cypress/no-unnecessary-waiting, cypress/unsafe-to-chain-command
```

Comment:
We should probably refactor the cypress tests to avoid the linting error, as it seems like a sound warning, but for now this eslint ignore is fine.